src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -305,12 +305,22 @@ protected void Stop()
                 var sb = new StringBuilder();
                 for (int i = 0; i < launchParams.Args.Length; i++)
```

Comment:
I vote we replace all this logic with `arguments = string.Join(launchParams.Args, " ")`

Reply:
Sounds good to me.